src/commands/shard.js

```diff
@@ -6,7 +6,7 @@ module.exports = async function shard (message, parent) {
   if (!message.data.args) return message.channel.send('Missing Arguments.')
   if (!parent.client.shard) return message.channel.send('Shard Manager not found.')
   const res = parent.client.shard.broadcastEval(message.data.args)
-  const result = await res.then(r => r).catch(e => e)
+  const result = await res.catch(e => e.toString())
```

Comment:
```suggestion
  const result = await res.then(r => r).catch(e => e)
```

Why are you making this change?

Reply:
```js
await (/* Promise-based Function */).then(r => r)
``` 
is equal to 
```js
await (/* Promise-based Function */)
``` and I’m stringifying error because ProcessManager is receiving error itself as instance of Error, and this is reason for not having a error message in Discord. The 'content' parameter needs to be string but Error is being passed.